.github/workflows/docker.yml

```diff
@@ -0,0 +1,115 @@
+name: docker
+
+on:
+  schedule:
+    - cron: '0 10 * * *' # everyday at 10am
```

Comment:
since we are publishing with fixed framework versions it would be sufficient to publish when merging into master, not every day, so coupling this to our unit test workflow

Reply:
That's mostly true, except for the Ray image, which is updated nightly.  Might be worth finding a way to exclude the others.